### ISSUE TITLE
Panel: does violate a aria-attribute rule

### ISSUE THREAD

**Issue by EugenFischer** [primefaces/primefaces#10653]
### Describe the bug

According Lighthouse Report and axe DevTool the the element which has aria-expanded="true" / "false" need to be a "control" element.
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role#aria-expanded
Additional to https://github.com/primefaces/primefaces/issues/9993 the div need an additional attribute role="button".

```
<div id="form1:panel1"
     class="ui-panel ui-widget ui-widget-content ui-corner-all"
     data-widget="panel"
     role="region"
     aria-labelledby="form1:panel1_header">

    <div id="form1:panel1_header"
         class="ui-panel-titlebar ui-widget-header ui-helper-clearfix ui-corner-all"
         role="button"
         aria-expanded="true"
         aria-controls="form1:panel1_content">
        <span class="ui-panel-title">Header</span>
        <a id="form1:panel1_toggler" href="#" class="ui-panel-titlebar-icon ui-corner-all ui-state-default"
           aria-label="Toggle Panel">
            <span class="ui-icon ui-icon-minusthick"/>
        </a>
    </div>

    <div id="form1:panel1_content"
         class="ui-panel-content ui-widget-content"
         aria-hidden="false">
        <p class="m-0">
            Lorem ipsum dolor sit amet ...
        </p>
    </div>

    <input id="form1:panel1_collapsed" name="form1:panel1_collapsed" type="hidden" autocomplete="off"
           value="false"/>
    <input id="form1:panel1_visible" name="form1:panel1_visible" type="hidden" autocomplete="off"
           value="true"/>
</div>
```

### Reproducer

1. navigate to https://www.primefaces.org/showcase/ui/panel/panel.xhtml
2. execute Lighthouse scan for Accessibility
3. analyze report -> [aria-*] attributes do not match their roles

### Expected behavior

no violation detected by Lighthouse / axe DevTool 

### PrimeFaces edition

None

### PrimeFaces version

13.0.0

### JSF implementation

Mojarra

### JSF version

2.3.16

### Java version

17

### Browser(s)

Chrome (116)

**Comment by melloware** [comment]
Can you review my PR to make sure it looks like you expect?

**Comment by melloware** [comment]
cc @timdoering
